run_script_cv_fgh: line 6: pythn: command not found
run_script_cv_fgh: line 6: pythn: command not found
run_script_cv_fgh: line 6: pythn: command not found
